eng/restore/runtime/runtime.depproj

```diff
@@ -17,6 +17,10 @@
       <Version>$(MicrosoftNETCoreRuntimeCoreCLRPackageVersion)</Version>
       <!-- Following is version used in release/uwp6.0 -->
       <Version Condition="'$(TargetFramework)'=='uap10.0.16299'">2.1.0-b-uwp6-25707-02</Version>
+      <Version Condition="'$(TargetFramework)'=='netcoreapp3.0'">3.0.0-preview9-19407-2</Version>
```

Comment:
Similar to above, we need an issue to replace this once we have an official build. It might make it more readable to factor these into properties so it's clear that they are for building against the 3.0 runtime.